tests/unit/test_elements.py

```diff
@@ -108,6 +108,41 @@ def test_outputs(self):
         assert "outputs" in yaml
         assert yaml["outputs"] == {"string-output": "foo", "expr-output": "${{ bar }}"}
 
+    def test_matrix(self):
```

Comment:
nitpick: Could you add a simple one or two liner docstring here (and below) to describe the test?

Reply:
I could, but none of the other tests do and I think it's clear what's being tested. As this is a fork of a public repo I'm inclined to stick to the upstream style more strictly than I otherwise would. That said, the upstream repo is unmaintained so maybe we should consider it ours now 🤔